src/librbd/ImageCtx.h

```diff
@@ -158,6 +158,7 @@ namespace librbd {
     EventSocket event_socket;
 
     ContextWQ *op_work_queue;
+    ContextWQ *pcache_op_work_queue; //chendi: used for multi process cache op
```

Comment:
Never mind -- I see that it is owned by the cache but is just being stored here. I'd say that shouldn't be needed.